package.json

```
@@ -0,0 +1,8 @@
+{
+  "dependencies": {
```

I think it would be more correct to have those as `devDependencies`. Even though it might not be relevant at all as we don't plan to publish this as a NPM package anyways.

```suggestion
  "devDependencies": {
```

---

Ok